src/sass/layouts/hero.scss

```diff
@@ -0,0 +1,416 @@
+  }
+  
+  
+  @media screen and (min-width: 768px) and (max-width: 1199px){
+    .coctails-section-coctailTitel{
+      margin-bottom: 62px;
+      width: 331px;
+      font-size: 32px;
+      line-height: 42px;
+    }
+  .container {
+      width: 768px;
+  }
+  .hero{
+    display: flex;
+    padding-top: 132px;
+    padding-bottom: 60px;
+  
+  &__box{
+    margin-right: auto;
+   
+   }
+   &-container{
+    width: 336px;
+    height: 465px;
+    background-size: cover;
+  background-image: url(../images/hero_tab.png);
+  transform: matrix(1, 0, 0, 1, 0,-65);
+  
+  }
+   
+  &-title{
+    font-weight: 600;
+    font-size: 26px;
+    line-height: 42px;
+  }
+  &-text{
+   
+  margin-top: 40;
+  
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 28px;
+    /* identical to box height, or 156% */
+    letter-spacing: 0.02em;
+  
+  }
+  
+  &-list {
+    margin-top: 40px;
+      width: 324px;
+      display: flex;
+      flex-wrap: wrap;
+      height: 75px;
+   
+    }
+ 
+    &-button{
+      width: 24px;
+      height: 25px;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 2.1%;
+      display: flex;
+  }
+  }
+  .coctails-section__coctails-img{
+    
+    width: 345px;
+    height: 380.74px;
+    background-size: cover;
+    background-image: url(../images/frame_desc.png);
+   
+   
+  }
+  }
+
+    @media screen and (max-width: 767px){
+      .coctails-section-coctailTitel{
+        margin-bottom: 63px;
+        width: 280px;
+        font-size: 22px;
+        line-height: 42px;
+      }
+      .container {
+          width: 320px;
+      }
+      .hero{
+        padding-top: 60px;
+        padding-bottom: 40px;
+    
+        &-list {
+          margin-top: 40px;
+          }
+          &-item{
+            width: 24px;
+            height: 25px;
+     
+          }
+          &-title{
+            margin-bottom: 28px;
+            font-weight: 600;
+            font-size: 22px;
+            line-height: 42px;
+          }
+          &-text{
+           
+          margin-top: 28px;
+          
+          font-weight: 500;
+          font-size: 14px;
+          line-height: 28px;
+            /* identical to box height, or 156% */
+            letter-spacing: 0.02em;
+            opacity: 0.8;
+          }
+        &-container{
+          width: 300px;
+          height: 368px;
+          background-size: cover;
+        background-image: url(../images/hero_tab.png);
+        transform: matrix(1, 0, 0, 1, 0, 0);
+        
+        }
+        &-div {
+          display: block;
+          display: flex;
+      align-items: center;
+      flex-direction: column;
+      
+          
+        justify-content: center;
+        }
+        &-boxList{
+          width: 93px;
+          height: 34px;
+          border: 1px solid $second-text-color;
+          border-radius: 4px;
+          justify-content: space-between;
+          display: flex;
+          flex-direction: row;
+           align-items: center; 
```

Comment:
TODO:

1) maybe line 302 is unnecessary? 'display: flex' puts elements in row
2) lines 300, 301, 303 replace with @mixin flex (space-between)